recipe/build.sh

```diff
@@ -109,6 +109,7 @@ if [[ ${cuda_compiler_version} != "None" ]]; then
     export GCC_HOST_COMPILER_PREFIX="$(dirname ${GCC})"
     # No obvious way to specify a different ld, so just link it to conda-forge's
     ln -s ${LD} $(dirname ${LD})/ld
+    ln -s ${AR} $(dirname ${LD})/ar
```

Comment:
This looks like a promising track! If you want to do all binutils

```
    ln -s ${AR} $(dirname ${AR})/ar
    ln -s ${AS} $(dirname ${AS})/as
    ln -s ${CXXFILT} $(dirname ${CXXFILT})/c++filt
    ln -s ${ELFEDIT} $(dirname ${ELFEDIT})/elfedit
    ln -s ${GPROF} $(dirname ${GPROF})/gprof
    ln -s ${LD_GOLD} $(dirname ${LD_GOLD})/ld.gold
    ln -s ${LD} $(dirname ${LD})/ld
    ln -s ${NM} $(dirname ${NM})/nm
    ln -s ${OBJCOPY} $(dirname ${OBJCOPY})/objcopy
    ln -s ${OBJDUMP} $(dirname ${OBJDUMP})/objdump
    ln -s ${RANLIB} $(dirname ${RANLIB})/ranlib
    ln -s ${READELF} $(dirname ${READELF})/readelf
    ln -s ${SIZE} $(dirname ${SIZE})/size
    ln -s ${STRINGS} $(dirname ${STRINGS})/strings
    ln -s ${STRIP} $(dirname ${STRIP})/strip
```
should be pretty comprehensive

Reply:
i could, but....... it seems i'm still hitting the -lsnappy and `-lprotobuf` errors even though conda-forge `ld` was used. Will add your list though.

Reply:
You do not need to do so by yourself. You can add `binutils` package, which has added symbolic links here: https://github.com/conda-forge/binutils-feedstock/blob/11da643035ec29d219d7d32a7c8bb6d6ef911d45/recipe/install_binutils_symlinks.sh#L8

Reply:
thanks!